spring-demo-ui/pom.xml

```diff
@@ -12,70 +12,36 @@
     <name>spring-demo-ui</name>
     <packaging>war</packaging>
 
-    <dependencyManagement>
```

Comment:
this was correct, explicitly importing a BOM as a normal dependency is not - BOMs are special